data-esnext.js

```diff
@@ -1179,6 +1179,7 @@ exports.tests = [
         babel: true,
         typescript: typescript.corejs,
         duktape20: false,
+        chrome60: true,
```

Comment:
Are you sure?

<img width="1037" alt="screen shot 2017-04-21 at 3 39 48 am" src="https://cloud.githubusercontent.com/assets/1010488/25269983/80cacf8c-2644-11e7-8bb2-cbb37783e556.png">